server/webserver.go

```diff
@@ -717,7 +701,11 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 				if !init {
 					return errors.New("Round not initialized")
 				}
-				err = s.LivepeerNode.Eth.ClaimEarnings(endRound)
+				currRound, err := s.LivepeerNode.Eth.CurrentRound()
+				if err != nil {
+					return err
+				}
+				_, err = s.LivepeerNode.Eth.ClaimEarnings(currRound)
```

Comment:
Since the other webserver endpoints for tx submission return after the tx is mined ([example](https://github.com/livepeer/go-livepeer/blob/master/server/webserver.go#L557)), I think it would be good to have that behavior here as well by calling `s.LivepeerNode.CheckTx()` with the transaction returned by `ClaimEarnings()` as the argument.